src/main.py

```diff
@@ -18,6 +19,7 @@ def main():
     cli_server = RemoteCLIServer()
     app = QtWidgets.QApplication([])
     app.setStyleSheet(Style.APP)
+    JoystickHandler()
```

Comment:
Why do we not store the reference here? Is it self-registering?

Reply:
because its a static class, only exists once, but i thought it should get initialized here